Account/FirefoxAccount.swift

```diff
@@ -190,13 +195,79 @@ open class FirefoxAccount {
             return "Not in a Token State: \(state?.label.rawValue ?? "Empty State")"
         }
     }
-
+    
+    public class FxAProfile {
+        open var avatarUrl: URL?
+        open var avatarImageData: Data?
```

Comment:
Sounds good I'll update. My original logic was that I didn't want to introduce view objects in the model class.